### PR TITLE
Cross-picked port of #3886

### PR DESCRIPTION
**Pull request by rtibbles**

### CODE REVIEW

**Comment by indirectlylit** [comment]
@lyw07 would you mind testing this?

**Comment by lyw07** [comment]
yes sure!

File: kolibri/logger/serializers.py [hunk 1]
```diff
@@ -49,11 +59,12 @@ class Meta:
 
     def get_pastattempts(self, obj):
         # will return a list of the latest 10 correct and hint_taken fields for each attempt.
-        return AttemptLog.objects.filter(masterylog__summarylog=obj.summarylog).values('correct', 'hinted').order_by('-start_timestamp')[:10]
+        return AttemptLog.objects.filter(masterylog__summarylog=obj.summarylog).values('correct', 'hinted', 'error').order_by('-start_timestamp')[:10]
```
Comment (lyw07) [inline kolibri/logger/serializers.py]: `error` field doesn't exist on the current AttemptLog model

**Review by lyw07 — CHANGES_REQUESTED** [review]

File: kolibri/logger/serializers.py (hunk 1 above)
Comment (rtibbles) [reply kolibri/logger/serializers.py]: Thanks, yeah, this doesn't exist in 0.9.

**Review by lyw07 — APPROVED** [review]
Looks good to me!

**Pull request merged**